runelite-client/src/main/java/net/runelite/client/plugins/microbot/util/inventory/Rs2Inventory.java

```diff
@@ -1289,13 +1371,13 @@ public static boolean isSlotFull(int slot) {
      * @param rs2Item The item to get the bounds for.
      * @return The bounding rectangle for the item's slot, or null if the item is not found.
      */
-    public static java.awt.Rectangle itemBounds(Rs2Item rs2Item) {
+    public static Rectangle itemBounds(Rs2Item rs2Item) {
         Widget inventory = getInventory();
 
         if (inventory == null) return null;
```

Comment:
What happens when null is returned, will the invoke crash?

Reply:
Shouldn't theoretically be possible based on the other checks before, but i pushed changes to handle null values just in case